CHANGES

```diff
@@ -50,6 +50,7 @@ Next release
 * ENH: ANTs JointFusion() (https://github.com/nipy/nipype/pull/1042)
 * ENH: Added csvReader() utility (https://github.com/nipy/nipype/pull/1044)
 * FIX: typo in nipype.interfaces.freesurfer.utils.py Tkregister2 (https://github.com/nipy/nipype/pull/1083)
+* FIX: SSHDataGrabber outputs now return full path to the grabbed/downloaded files.
```

Comment:
add URL : https://github.com/nipy/nipype/pull/1086

we are trying to create a direct link back from the changes so people can see the exact changes.